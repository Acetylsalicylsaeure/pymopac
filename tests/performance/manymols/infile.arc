

                     SUMMARY OF  PM7 CALCULATION

                                                       MOPAC v22.1.1 Linux
                                                       Wed Sep 25 20:54:25 2024

           Empirical Formula: C3 H8  =    11 atoms

 PM7
 #



     GEOMETRY OPTIMISED USING EIGENVECTOR FOLLOWING (EF).     
     SCF FIELD WAS ACHIEVED                                   

          HEAT OF FORMATION       =        -23.08806 KCAL/MOL =     -96.60044 KJ/MOL
          GRADIENT NORM           =          0.50215          =       0.15140 PER ATOM
          DIPOLE                  =          0.05744 DEBYE   POINT GROUP:  C2v 
          NO. OF FILLED LEVELS    =         10
          IONIZATION POTENTIAL    =         11.392873 EV
          HOMO LUMO ENERGIES (EV) =        -11.393  4.300
          MOLECULAR WEIGHT        =         44.0962
          COSMO AREA              =         95.70 SQUARE ANGSTROMS
          COSMO VOLUME            =         79.01 CUBIC ANGSTROMS

          MOLECULAR DIMENSIONS (Angstroms)

            Atom       Atom       Distance
            H     9    H     5     4.33234
            H     8    H    10     2.80573
            H     7    H     4     2.74431
          SCF CALCULATIONS        =         14
          WALL-CLOCK TIME         =      0.016 SECONDS
          COMPUTATION TIME        =      0.011 SECONDS


          FINAL GEOMETRY OBTAINED
 PM7
 #

  C     1.21434898 +1  -0.16354246 +1  -0.37967460 +1
  C     0.04063375 +1  -0.18590379 +1   0.60004914 +1
  C    -1.24460409 +1   0.30257465 +1  -0.06913778 +1
  H     1.40047736 +1   0.84825447 +1  -0.75575811 +1
  H     2.13627027 +1  -0.51578903 +1   0.09534840 +1
  H     1.02291967 +1  -0.80534161 +1  -1.24669714 +1
  H    -0.10445328 +1  -1.21038149 +1   0.99290905 +1
  H     0.27069445 +1   0.44489525 +1   1.47965666 +1
  H    -2.08730951 +1   0.28705274 +1   0.63010630 +1
  H    -1.51268725 +1  -0.32609419 +1  -0.92540389 +1
  H    -1.13777543 +1   1.32892261 +1  -0.43701130 +1

