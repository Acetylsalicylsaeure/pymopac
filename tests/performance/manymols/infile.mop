PM7
#

C      1.210587   -0.163303   -0.377666
C      0.039837   -0.183317    0.590483
C     -1.240651    0.301644   -0.067948
H      1.396895    0.850202   -0.747135
H      2.119691   -0.516279    0.119160
H      1.019449   -0.812761   -1.237946
H     -0.106100   -1.202425    0.965357
H      0.269289    0.451489    1.453491
H     -2.067917    0.278024    0.648271
H     -1.509261   -0.333119   -0.918438
H     -1.131819    1.329844   -0.427628
