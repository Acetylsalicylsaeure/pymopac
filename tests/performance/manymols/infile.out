 *******************************************************************************
 **                                                                           **
 **                              MOPAC v22.1.1                                **
 **                                                                           **
 *******************************************************************************
 **          Digital Object Identifier (DOI): 10.5281/zenodo.6511958          **
 **    Visit the DOI location for information on how to cite this program.    **
 *******************************************************************************

                              PM7 CALCULATION RESULTS

 *******************************************************************************
 *  CALCULATION DONE:                                Wed Sep 25 20:54:25 2024  *
 *  PM7        - The PM7 Hamiltonian to be used
 *  T=         - A TIME OF 172800.0 SECONDS REQUESTED
 *  DUMP=N     - RESTART FILE WRITTEN EVERY 7200.0 SECONDS REQUESTED
 *******************************************************************************
PM7
 #

   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)

     1       C          1.21058700  *  -0.16330300  *  -0.37766600  *
     2       C          0.03983700  *  -0.18331700  *   0.59048300  *
     3       C         -1.24065100  *   0.30164400  *  -0.06794800  *
     4       H          1.39689500  *   0.85020200  *  -0.74713500  *
     5       H          2.11969100  *  -0.51627900  *   0.11916000  *
     6       H          1.01944900  *  -0.81276100  *  -1.23794600  *
     7       H         -0.10610000  *  -1.20242500  *   0.96535700  *
     8       H          0.26928900  *   0.45148900  *   1.45349100  *
     9       H         -2.06791700  *   0.27802400  *   0.64827100  *
    10       H         -1.50926100  *  -0.33311900  *  -0.91843800  *
    11       H         -1.13181900  *   1.32984400  *  -0.42762800  *



          CARTESIAN COORDINATES 

    NO.       ATOM           X           Y           Z

     1         C          1.2106     -0.1633     -0.3777
     2         C          0.0398     -0.1833      0.5905
     3         C         -1.2407      0.3016     -0.0679
     4         H          1.3969      0.8502     -0.7471
     5         H          2.1197     -0.5163      0.1192
     6         H          1.0194     -0.8128     -1.2379
     7         H         -0.1061     -1.2024      0.9654
     8         H          0.2693      0.4515      1.4535
     9         H         -2.0679      0.2780      0.6483
    10         H         -1.5093     -0.3331     -0.9184
    11         H         -1.1318      1.3298     -0.4276

 General Reference for PM7:
 "Optimization of Parameters for Semiempirical Methods VI: More Modifications to the 
 NDDO Approximations and Re-optimization of Parameters", J. J. P. Stewart, J. Mol. Mod., 1:32, 19 (2013)
 https://link.springer.com/article/10.1007/s00894-012-1667-x

           Empirical Formula: C3 H8  =    11 atoms



      MOLECULAR POINT GROUP   :   C2v 


      RHF CALCULATION, NO. OF DOUBLY OCCUPIED LEVELS = 10


          Geometry optimization using EF

          DIAGONAL MATRIX USED AS START HESSIAN

 CYCLE:     1 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:    16.029 HEAT: -22.73066
 CYCLE:     2 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:    20.757 HEAT: -22.78903
 CYCLE:     3 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:    16.060 HEAT: -22.93499
 CYCLE:     4 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:    11.463 HEAT: -22.98541
 CYCLE:     5 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:    14.374 HEAT: -22.99729
 CYCLE:     6 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:     7.757 HEAT: -23.04973
 CYCLE:     7 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:     5.878 HEAT: -23.06841
 CYCLE:     8 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:     3.437 HEAT: -23.07937
 CYCLE:     9 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:     4.381 HEAT: -23.07895
 CYCLE:    10 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:     2.065 HEAT: -23.08527
 CYCLE:    11 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:     1.159 HEAT: -23.08721
 CYCLE:    12 TIME:   0.000 TIME LEFT:  2.00D  GRAD.:     0.502 HEAT: -23.08806

     GRADIENT =  0.50215 IS LESS THAN CUTOFF =  1.00000



 -------------------------------------------------------------------------------
 PM7
 #



     GEOMETRY OPTIMISED USING EIGENVECTOR FOLLOWING (EF).     
     SCF FIELD WAS ACHIEVED                                   


                              PM7 CALCULATION
                                                       MOPAC v22.1.1 Linux
                                                       Wed Sep 25 20:54:25 2024




          FINAL HEAT OF FORMATION =        -23.08806 KCAL/MOL =     -96.60044 KJ/MOL


          COSMO AREA              =         95.70 SQUARE ANGSTROMS
          COSMO VOLUME            =         79.01 CUBIC ANGSTROMS

          GRADIENT NORM           =          0.50215          =       0.15140 PER ATOM
          IONIZATION POTENTIAL    =         11.392873 EV
          HOMO LUMO ENERGIES (EV) =        -11.393  4.300
          NO. OF FILLED LEVELS    =         10
          MOLECULAR WEIGHT        =         44.0962         POINT GROUP:  C2v 

          MOLECULAR DIMENSIONS (Angstroms)

            Atom       Atom       Distance
            H     9    H     5     4.33234
            H     8    H    10     2.80573
            H     7    H     4     2.74431


          SCF CALCULATIONS        =         14

          WALL-CLOCK TIME         =      0.016 SECONDS
          COMPUTATION TIME        =      0.010 SECONDS




   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)

     1       C          1.21434898  *  -0.16354246  *  -0.37967460  *
     2       C          0.04063375  *  -0.18590379  *   0.60004914  *
     3       C         -1.24460409  *   0.30257465  *  -0.06913778  *
     4       H          1.40047736  *   0.84825447  *  -0.75575811  *
     5       H          2.13627027  *  -0.51578903  *   0.09534840  *
     6       H          1.02291967  *  -0.80534161  *  -1.24669714  *
     7       H         -0.10445328  *  -1.21038149  *   0.99290905  *
     8       H          0.27069445  *   0.44489525  *   1.47965666  *
     9       H         -2.08730951  *   0.28705274  *   0.63010630  *
    10       H         -1.51268725  *  -0.32609419  *  -0.92540389  *
    11       H         -1.13777543  *   1.32892261  *  -0.43701130  *

                             CARTESIAN COORDINATES

   1    C        1.214348984    -0.163542456    -0.379674601
   2    C        0.040633747    -0.185903788     0.600049141
   3    C       -1.244604094     0.302574646    -0.069137780
   4    H        1.400477361     0.848254474    -0.755758108
   5    H        2.136270273    -0.515789032     0.095348404
   6    H        1.022919672    -0.805341610    -1.246697139
   7    H       -0.104453277    -1.210381485     0.992909054
   8    H        0.270694454     0.444895255     1.479656661
   9    H       -2.087309514     0.287052735     0.630106300
  10    H       -1.512687250    -0.326094186    -0.925403885
  11    H       -1.137775425     1.328922606    -0.437011301


           Empirical Formula: C3 H8  =    11 atoms



      MOLECULAR POINT GROUP   :   C2v 


                  EIGENVALUES  
 -28.15963 -23.78290 -20.12264 -15.81624 -15.09317 -13.90740 -13.15121 -12.00276
 -11.97326 -11.39287   4.29974   4.65523   5.50974   5.89454   5.94727   5.98393
   6.23846   6.66514   7.06308   7.13278


              NET ATOMIC CHARGES AND DIPOLE CONTRIBUTIONS

  ATOM NO.   TYPE          CHARGE      No. of ELECS.   s-Pop       p-Pop       
    1          C          -0.438845        4.4388     1.05217     3.38667
    2          C          -0.229601        4.2296     1.05086     3.17874
    3          C          -0.438901        4.4389     1.05221     3.38669
    4          H           0.142184        0.8578     0.85782
    5          H           0.138624        0.8614     0.86138
    6          H           0.142203        0.8578     0.85780
    7          H           0.130677        0.8693     0.86932
    8          H           0.130662        0.8693     0.86934
    9          H           0.138598        0.8614     0.86140
   10          H           0.142202        0.8578     0.85780
   11          H           0.142196        0.8578     0.85780
 DIPOLE           X         Y         Z       TOTAL
 POINT-CHG.     0.001    -0.007     0.021     0.022
 HYBRID        -0.005     0.023    -0.076     0.079
 SUM           -0.004     0.017    -0.055     0.057


          ATOMIC ORBITAL ELECTRON POPULATIONS

     Atom    s        px        py        pz   
    1  C   1.05217   1.09168   1.17364   1.12135
    2  C   1.05086   0.98256   1.13948   1.05669
    3  C   1.05221   1.07754   1.16145   1.14770
    4  H   0.85782
    5  H   0.86138
    6  H   0.85780
    7  H   0.86932
    8  H   0.86934
    9  H   0.86140
   10  H   0.85780
   11  H   0.85780

 **********************
 *                    *
 * JOB ENDED NORMALLY *
 *                    *
 **********************



 TOTAL JOB TIME:             0.02 SECONDS

 == MOPAC DONE ==
